CONTRIBUTING.md

```diff
@@ -0,0 +1,246 @@
+# Contributing to our Blog
+
+Authors who are informed and passionate about technology and IT write technical
+blogs that range from high-level, conceptual overviews to deep dives with
+code snippets. They all offer insight into solving problems and getting the most
+out of applications and platforms. Many also add a personal perspective to the
+conversation. These authors share their journeys, trials, and triumphs.
+
+First, thanks for your interest in contributing and helping us craft quality
+content for our official Technical Blog. Second, in order to make contributing
+a pleasant experience while maintaining a visual and consistent content standard,
+complete these steps before writing and submitting a blog entry for publishing.
+
+For ideas of topics to blog about, check out the end of this file!
+
+#### Preparing for your blog entry
+
+1. Fork this repo (https://github.com/rackerlabs/technical-blog), then
+   clone your fork.
+
+2. Before you start working, make sure your local content is up-to-date and
+   merged with the right upstream branch:
+
+      ```bash
+         git remote add upstream git@github.com:rackerlabs/docs-developer-blog.git
+         git checkout master
+         git fetch upstream
+         git checkout -b name-of-your-branch
+         git merge upstream/master
+      ```
+
+3. Create a new folder inside the `/hugo/content/blog` directory with the following naming
+   convention: `YYYY-MM-DD-title-of-your-post`, where `YYYY-MM-DD` is the
+   date you want your entry to be published.
+
+4. Within the newly created folder, create a new file with the following naming convention:
+   `index.md`.The published URL for your post becomes `https://docs.rackspace.com/blog/title-of-your-post/` when published.
+
+**Note:** If your post has images, you should place the image files within the newly created
+folder.
+
+#### Formatting your post
+
+The post should contain front-matter, an excerpt, and the actual content.
+
+##### Front-matter
+
+Add Jekyll front-matter (or metadata) to the top of the file you created in
```

Comment:
remove Jekyll

but this should be handled automatically by an archetype. the writer will run a command to generate the directory and index.md with initial metadata. Check with @codebyravi .

Reply:
I chose to comment out this section out for now.

Reply:
I'd love to look at the front-matter/metadata options and help decide what we need to have to create the best possible experience for everyone-readers and writers.